src/components/BottomBar/index.js

```diff
@@ -60,13 +46,13 @@ export default class BottomBar extends Component {
 const BlockInfo = withStyles(styles)(({ classes, blockNum, blockTime }) => (
   <Grid item xs={12} md={6} className={classes.bottomBarBlockInfoWrapper}>
     <Typography variant="body1">
-      <span className={classes.bottomBarBlockNum}><FormattedMessage id="bottomBar.blockNum" defaultMessage="Current Block Number" />: {blockNum}</span>
-      <FormattedMessage id="bottomBar.blockTime" defaultMessage="Current Block Time" />: {blockTime ? getShortLocalDateTimeString(blockTime) : ''}
+      <span className={classes.bottomBarBlockNum}><FormattedMessage id="bottomBar.blockNum" defaultMessage="Current Block Number" />:&nbsp;{blockNum}</span>
```

Comment:
can we do this as a string template? `: ${blockNum}`

Reply:
good idea